packages/compartment-mapper/src/compartment-map.js

```diff
@@ -140,13 +140,15 @@ const assertModule = (allegedModule, path, url) => {
     `${path} must be an object, got ${allegedModule} in ${q(url)}`,
   );
 
-  const { compartment, module, location, parser, exit } = moduleDescriptor;
+  const { compartment, module, location, parser, exit, deferredError } = moduleDescriptor;
   if (compartment !== undefined || module !== undefined) {
     assertCompartmentModule(moduleDescriptor, path, url);
   } else if (location !== undefined || parser !== undefined) {
     assertFileModule(moduleDescriptor, path, url);
   } else if (exit !== undefined) {
     assertExitModule(moduleDescriptor, path, url);
+  } else if (deferredError !== undefined) {
+    //it's ok, trust me
```

Comment:
Is it a string?

Reply:
It's a string containing the error message. I'll add an assertion eventually.